VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.ts

```diff
@@ -27,6 +30,10 @@ export class JudgeWaitingRoomComponent extends WaitingRoomBaseComponent implemen
     showAudioRecordingAlert = false;
     expanedPanel = true;
 
+    consultationAccepted$: Subscription;
+
+    @ViewChild(SelectMediaDevicesComponent) selectMediaDevices: SelectMediaDevicesComponent;
```

Comment:
Why do you need a reference to the select media devices component here? If you're looking to call the submit function, it would be cleaner to not create a strong depenedency between the two. If you're looking to close the change device modal when a person is moved to consultation have a look at `handleParticipantStatusChange` in `WaitingRoomBaseComponent`

Reply:
it's for close dialog and run dialog method that close streams on dialog form